### DISCUSSION TITLE
Enabling video hardware acceleration on integrated Intel UHD Graphics 620 GPU and debian on chromium based browsers

### DISCUSSION THREAD

**Discussion by pierro78** [Alex313031/thorium#39] [Show and tell]
see https://www.reddit.com/r/ChromiumBrowser/comments/v3xkd9/enabling_video_hardware_acceleration_on_intel_uhd/

( I thought that maybe this could help my fanless laptop stay cool and maybe interest other people ... )

**Comment by Alex313031** [comment]
@pierro78 Thorium has had patches for this for a while now. So just responding so we can close unless theres something else you wanna add.
